src/main/java/uk/gov/register/presentation/dao/Entry.java

```diff
@@ -0,0 +1,38 @@
+package uk.gov.register.presentation.dao;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.sql.Timestamp;
```

Comment:
this is a mutable object class. is there an immutable alternative? Perhaps from joda?